app.rb

```diff
@@ -0,0 +1,19 @@
+require 'pg'
+require 'sinatra/base'
+require './lib/peep.rb'
+
+class Chitter < Sinatra::Base
+
+  get '/' do
+    @peeps = Peep.all
+    erb(:index)
+  end
+
+  post '/postpeep' do
+    Peep.create(peep: params["peep"])
```

Comment:
Bit nit picky but convention is to user params[:peep]. Using a symbol instead of string makes sure that it's going to always be unmutable.